src/ophyd_async/epics/_backend/common.py

```diff
@@ -1,12 +1,33 @@
+from dataclasses import dataclass
 from enum import Enum
 from typing import Dict, Optional, Tuple, Type
 
-_common_meta = {
+common_meta = {
     "units",
     "precision",
 }
 
 
+@dataclass
+class LimitPair:
+    high: float | None = None
+    low: float | None = None
+
+    def __bool__(self):
+        return self.high is not None or self.low is not None
+
+
+@dataclass
+class Limits:
+    alarm: LimitPair
+    control: LimitPair
+    display: LimitPair
+    warning: LimitPair
```

Comment:
@evalott100 does this match with how you want to do limits in the event-model?

Reply:
It can do, though if you plan to validate these against an event-model schema then I think it's better to make `LimitPair` a `TypedDict`.

Reply:
Adjusted LimitPair and Limits to be TypedDict, made high, low float with nan for when not set, so's to be slightly more predictable (?)